cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -167,6 +167,11 @@ func (client *autoscalingGceClientV1) FetchMigTargetSize(migRef GceRef) (int64,
 func (client *autoscalingGceClientV1) FetchMigBasename(migRef GceRef) (string, error) {
 	registerRequest("instance_group_managers", "get")
 	igm, err := client.gceService.InstanceGroupManagers.Get(migRef.Project, migRef.Zone, migRef.Name).Do()
+	if err, ok := err.(*googleapi.Error); ok {
```

Comment:
nit: you can make this
if err, ok := err.(*googleapi.Error); ok && err.Code == http.StatusNotFound {
...
}

Reply:
Done